website/sidebars.json

```diff
@@ -153,19 +150,26 @@
           "querying/dimensionspecs",
           "querying/aggregations",
           "querying/post-aggregations",
-          "misc/math-expr",
+          "querying/math-expr",
           "querying/having",
           "querying/limitspec",
           "querying/topnmetricspec",
           "querying/sorting-orders",
           "querying/virtual-columns",
-          "development/geo"
+          "querying/geo"
         ]
       }
     ],
+    "API reference":[
+      "multi-stage-query/api",
```

Comment:
i notice that docs on the file system don't match up with this. will we be making that change in a later iteration?

Reply:
I'll update the file system.